komodo-core/src/main/resources/org/komodo/repository/local-repository-config.json

```diff
@@ -6,13 +6,6 @@
     },
     "node-types" :
     [
-    	"config/dv.cnd",
-        "config/xmi.cnd",
-        "config/med.cnd",
-        "config/mmcore.cnd",
-        "config/relational.cnd",
-        "config/vdb.cnd",
-        "config/TeiidSql.cnd",
```

Comment:
TeiidSql.cnd is loaded by the sequencer that needs it so it does not have to be loaded via the config file.